backend/database/generatePasswords.py

```diff
@@ -0,0 +1,41 @@
+import random
+import string
+import csv
+
+'''
+Generates 20 passwords between 10-24 characters that contain:
+- at least 1 lowercase letter
+- at least 1 uppercase letter
+- at least 1 digit
+- 1 special character
+
+since when presented with these requirements, most users choose 1 special character and fill in the rest of the characters with letters and digits. Writes to passwords to a csv file.
+'''
+passwords, i = set(), 0
+
+while i < 20:
```

Comment:
**suggestion**: Make this a function: `generate_passwords(num_passwords, min_chars, max_chars) -> set[str]`. Then you don't have to hardcode changes to the values.